src/commands/interactive/start-interactive-mode.ts

```diff
@@ -0,0 +1,74 @@
+import { CmdGlobalOptions } from "../cmd-options.ts";
+import { exploreInteractive } from "./untagged-tenants.ts";
+import { listTenantAction, listTenantsCostAction } from "../tenant.command.ts";
+import { Select } from "../../deps.ts";
+import { OutputFormat } from "../../presentation/output-format.ts";
+import { interactiveDate } from "./inputInteractiveDate.ts";
+
+export async function startInteractiveMode(options: CmdGlobalOptions) {
+  console.clear();
+  const interactivehelp =
```

Comment:
```suggestion
  const interactiveHelp =
```
using camel case often makes variables easier to read.

Reply:
was not fixed